src/components/SearchModal/CurrencyList.tsx

```diff
@@ -3,29 +3,34 @@ import { FixedSizeList } from 'react-window'
 import { Text } from 'rebass'
 import styled from 'styled-components'
 import { t, Trans } from '@lingui/macro'
+import { Star } from 'react-feather'
 
 import { Currency, CurrencyAmount, Token } from '@kyberswap/ks-sdk-core'
-import { useActiveWeb3React } from '../../hooks'
-import { useCombinedActiveList } from '../../state/lists/hooks'
-import { useCurrencyBalances } from '../../state/wallet/hooks'
-import { TYPE } from '../../theme'
-import { useIsUserAddedToken } from '../../hooks/Tokens'
-import Column from '../Column'
-import { RowFixed, RowBetween } from '../Row'
-import CurrencyLogo from '../CurrencyLogo'
-import { MouseoverTooltip } from '../Tooltip'
-import { MenuItem } from './styleds'
-import Loader from '../Loader'
-import { isTokenOnList, isAddress } from '../../utils'
-import ImportRow from './ImportRow'
+import { useActiveWeb3React } from 'hooks'
+import { useCombinedActiveList } from 'state/lists/hooks'
+import { useCurrencyBalances } from 'state/wallet/hooks'
+import { TYPE } from 'theme'
+import { useIsUserAddedToken } from 'hooks/Tokens'
+import { isTokenOnList, isAddress } from 'utils'
 import { LightGreyCard } from 'components/Card'
-import TokenListLogo from '../../assets/svg/tokenlist.svg'
+import TokenListLogo from 'assets/svg/tokenlist.svg'
 import QuestionHelper from 'components/QuestionHelper'
 import useTheme from 'hooks/useTheme'
 import { useCurrencyConvertedToNative } from 'utils/dmm'
 import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
 import { nativeOnChain } from 'constants/tokens'
 
+import { RowFixed, RowBetween } from '../Row'
+import Column from '../Column'
+import CurrencyLogo from '../CurrencyLogo'
+import { MouseoverTooltip } from '../Tooltip'
+import Loader from '../Loader'
+import ImportRow from './ImportRow'
+import { ButtonEmpty } from 'components/Button'
+import { useDispatch, useSelector } from 'react-redux'
+import { AppState } from 'state'
+import { toggleFavoriteToken } from 'state/user/actions'
```

Comment:
Cool bro, you did this manually or do you have any tool? :))

Reply:
Manually bro 😁